install_config/cluster_metrics.adoc

```diff
@@ -306,15 +340,15 @@ ifdef::openshift-enterprise[]
 ====
 endif::[]
 
-
-You will need to save your completed file with the file name *_metrics.yaml_*.
+Copy this file to another directory with the file name
+*_metrics-deployer.yaml_*.
```

Comment:
Is there any reason why we are telling people to copy this file somewhere and then deploy it? Can they not just deploy it from that location directly?